extensions/bindy/runtime/src/main/resources/META-INF/quarkus-extension.yaml

```diff
@@ -22,7 +22,7 @@
 #
 ---
 name: "Camel Bindy"
-description: "Marshal and unmarshal Java beans from and to flat payloads (such as CSV, delimited, fixed length formats, or FIX messages)"
+description: "Marshal and unmarshal between POJOs and Comma separated values (CSV) format using Camel Bindy Marshal and unmarshal between POJOs and fixed field length format using Camel Bindy Marshal and unmarshal between POJOs and key-value pair (KVP) format using Camel Bindy"
```

Comment:
It looks bindy is also modified.

Reply:
Similar issue for SnakeYAML.

Reply:
I've noticed it. It would be probably better, to move generated changes in bindy extension into separate commit (bindy's description was changed several days ago - https://github.com/apache/camel/blame/master/components/camel-bindy/src/main/docs/bindy-dataformat.adoc#L5)

Reply:
I've moved bot snakeYAML and bindy changes into separate commit